src/main/res/values/strings.xml

```diff
@@ -120,47 +120,47 @@
   <string name="saving_details">Saving details</string>
   <string name="screenshots">Screenshots</string>
   <string name="search">Search</string>
-  <string name="select_the_mirror">Select the mirror</string>
-  <string name="show_more">Show more</string>
-  <string name="show_older_releases">Show older releases</string>
+  <string name="select_the_mirror">Select a mirror</string>
+  <string name="show_more">More</string>
```

Comment:
"more" is too short in this context. Also "show older versions" uses word "more". Let's keep is as it is.

Reply:
I think just a "+" would suffice. What error is "More" making you do?

Reply:
"+" is even shorter. "Show more" is displayed to expand the full description, it's just a wide button below the text block.